### PR TITLE
CVE-2016-1709 and CVE-2013-2926

### PR DESCRIPTION
**Pull request by Primalink**
Updated CVE-2016-1709 and CVE-2013-2926.

mjc2649

### CODE REVIEW

File: cves/CVE-2016-1709.yml [hunk 1]
```diff
@@ -16,24 +16,28 @@ description_instructions: |
   expressions" is fine, and security phrases like "invalid write" are fine to
   keep too.
 description: 
+  Attackers could cause a Heap-based buffer overflow in sfntly (sfntly is a program that lets users create and edit custom text fonts). The buffer overflow could be used for denial of service attacks.
```
Comment (andymeneely) [inline cves/CVE-2016-1709.yml]: I'm wondering if your Yaml syntax error is coming from here.

Comment (Primalink) [inline cves/CVE-2016-1709.yml]: I deleted my description but it didn't affect the Yaml syntax error

File: cves/CVE-2013-2926.yml [hunk 2]
```diff
@@ -33,11 +32,9 @@ fixes_vcc_instructions: |
...
-- :commit: e1524692d362e607e806569147096dfb8c38cb6a
-  :note: ''
-- :commit: 24c7296e95a3f2518d9a84d262d806a87a07099a
-  :note: ''
-vccs: []
+- e1524692d362e607e806569147096dfb8c38cb6a
+- 24c7296e95a3f2518d9a84d262d806a87a07099a
+vccs: [7950ccde9e0ac9bca45c9b3a2002ad5776d47352]
 upvotes_instructions: |
   For the first round, ignore this upvotes number.
```
Comment (cgv2921) [inline cves/CVE-2013-2926.yml]: I'm giving this CVE 2 upvotes.  It seems like a standard vulnerability, so it didn't interest me much.

File: cves/CVE-2013-2926.yml [hunk 3]
```diff
@@ -177,3 +174,10 @@ mistakes:
     every one. Write a thoughtful entry here that those ing the software
```
Comment (cgv2921) [inline cves/CVE-2013-2926.yml]: In the "lessons" section, I feel you may want to write something for "least_privilege", as this is a vulnerability with a user-assisted exploit, that may be applicable.

**Pull request merged**